scene/gui/link_button.cpp

```diff
@@ -84,6 +96,7 @@ void LinkButton::_notification(int p_what) {
 						color = get_color("font_color");
 
 					do_underline = underline_mode != UNDERLINE_MODE_NEVER;
+					OS::get_singleton()->shell_open(link);
```

Comment:
This shouldn't be handled in the draw call, it should be in a callback connected to the BaseButton's `pressed` signal.